python/src/aac/io/parser/_yaml.py

```diff
@@ -55,7 +56,7 @@ def parse_yaml(source: str, content: str) -> list[dict]:
         If the model does not have (at least) a "name" field, a ParserError is raised.
     """
     try:
-        models = list(load_all(content, Loader=SafeLoader))
+        models = [model for model in load_all(content, Loader=SafeLoader) if model]
```

Comment:
Woo iterate and if checks for the win to evaluate empty/invalid instances! Good job!